packages/auth/src/accessToken/routes.ts

```diff
@@ -101,7 +113,15 @@ async function revokeToken(
   deps: ServiceDependencies,
   ctx: RevokeContext
 ): Promise<void> {
-  await deps.accessTokenService.revoke(ctx.accessToken.id)
+  const accessToken = await deps.accessTokenService.revoke(ctx.accessToken.id)
+
+  deps.logger.debug(
+    {
+      ...generateRouteLogs(ctx),
+      accessToken
```

Comment:
should we be logging out tokens such as accessToken?

Reply:
Probably not

Reply:
Does it make any difference that its a debug log? I guess we can say these _can_ but _shouldnt_ be logged in production. Had the same thought about `grant.continueToken` elsewhere.